docs/release-notes/NuGet-6.10.md

```diff
@@ -0,0 +1,178 @@
+
+* ResolvedDependencyKey should be struct to reduce memory allocations - [#13138](https://github.com/NuGet/Home/issues/13138)
+
+* [DCR]: Central Package Management - Respect .props file as a way to opt-in to the feature. - [#11834](https://github.com/NuGet/Home/issues/11834)
+
+### Issues fixed in this release
+
+* Stop using JObject in assets file reading to reduce allocations. [#12715](https://github.com/NuGet/Home/issues/12715)
+
+* SourceRepository.GetResourceAsync should be cancellable. - [#13234](https://github.com/NuGet/Home/issues/13234)
+
+* CreateWalkAsync should not be recursive - [#13222](https://github.com/NuGet/Home/issues/13222)
+
+* ProjectModel.HashObjectWriter.OnFlush is using a SHA512 hash versus a cheaper hash which seems like overkill - [#13214](https://github.com/NuGet/Home/issues/13214)
+
+* Owner package metadata is an array in JSON but a string in Protocol types - [#13186](https://github.com/NuGet/Home/issues/13186)
+
+* dotnet package search --verbosity detailed output table too wide - [#13162](https://github.com/NuGet/Home/issues/13162)
+
+* Calls in SetWarningProperties() have allocation overhead due to multiple enumeration - [#13151](https://github.com/NuGet/Home/issues/13151)
+
+* Use of ConcurrentStack in object pool implementation creates large amounts of allocations - [#13147](https://github.com/NuGet/Home/issues/13147)
+
+* IsBestVersion boxes HashSet<T> enumerators resulting in lots of allocation overhead during restore. - [#13146](https://github.com/NuGet/Home/issues/13146)
+
+* Deprecate NuGet.exe list in favor of NuGet.exe search - [#7912](https://github.com/NuGet/Home/issues/7912)
+
+* [Dotnet Package Search] An unhandled exception is thrown when searching with “--verbosity detailed” and “--format json” - [#13300](https://github.com/NuGet/Home/issues/13300)
+
+* `dotnet package search` crashes on .NET 9 preview 2 nightly - [#13286](https://github.com/NuGet/Home/issues/13286)
+
+* Use the StringBuilderPool rather than allocating a new StringBuilder - [#13285](https://github.com/NuGet/Home/issues/13285)
+
+* Pass in an appropriate size for List<T> - [#13284](https://github.com/NuGet/Home/issues/13284)
+
+* JsonTextWriter allocates a large number strings - [#13283](https://github.com/NuGet/Home/issues/13283)
+
+* Usage of StringBuilder.Append() allocates when appending an int - [#13282](https://github.com/NuGet/Home/issues/13282)
+
+* Unnecessary boxing of struct enumerators - [#13281](https://github.com/NuGet/Home/issues/13281)
+
+* Process objects should be disposed so that the finalizer doesn't run - [#13280](https://github.com/NuGet/Home/issues/13280)
+
+* Avoid boxing enumerators for collections - [#13279](https://github.com/NuGet/Home/issues/13279)
+
+* Display a vulnerability info bar when there's vulnerabilities in packages.config based projects. - [#13271](https://github.com/NuGet/Home/issues/13271)
```

Comment:
This is probably a feature?

Reply:
I agree. Moved this issue under `What's New` section.